### ISSUE TITLE
Option click vs cmd click

### ISSUE THREAD

**Issue by maschwenk** [zed-industries/zed#8787]
### Check for existing issues

- [X] Completed

### Describe the bug / provide steps to reproduce it

In VSCode, my goto definition and goto file when in the terminal seems bound to "option + click"
In Zed this all seems bound to "cmd + click"

It's unclear if this is configurable or not. It seems there's a lot of Issues filed around adding functionality to "cmd + click" but cannot seem to find anything about changing the modifier.

### Environment

Zed: v0.124.8 (Zed)
OS: macOS 14.3.0
Memory: 32 GiB
Architecture: aarch64

### If applicable, attach your `~/Library/Logs/Zed/Zed.log` file to this issue.
If you only need the most recent lines, you can run the `zed: open log` command palette action to see the last 1000.

_No response_

**Comment by maschwenk** [comment]
> It seems there's a lot of Issues filed around adding functionality to "cmd + click" but cannot seem to find anything about changing the modifier.

To be clear I don't even know if VSCode makes this configurable or not. I can't seem to find anything that suggests that.

**Comment by wottpal** [comment]
Did you found a solution/setting for this @maschwenk?

**Comment by maschwenk** [comment]
Nope! And I keep bouncing off of Zed because I find it hard to adjust to that

**Comment by mrdziuban** [comment]
I came across this too because I'm accustomed to using "cmd + click" to add another cursor. It would be great if I could swap these in Zed

**Comment by chanmathew** [comment]
Same here!
